test/libsolidity/semanticTests/builtinFunctions/eof/keccak256_packed_complex_types.sol

```diff
@@ -0,0 +1,17 @@
+contract C {
+    uint120[3] x;
+    function f() public returns (bytes32 hash1, bytes32 hash2, bool hash3NonZero) {
+        uint120[] memory y = new uint120[](3);
+        x[0] = y[0] = uint120(type(uint).max - 1);
+        x[1] = y[1] = uint120(type(uint).max - 2);
+        x[2] = y[2] = uint120(type(uint).max - 3);
+        hash1 = keccak256(abi.encodePacked(x));
+        hash2 = keccak256(abi.encodePacked(y));
+        hash3NonZero = bytes32(0) != keccak256(abi.encodePacked(this.f));
+    }
+}
+// ====
+// EVMVersion: >=prague
+// bytecodeFormat: >=EOFv1
```

Comment:
`EVMVersion: >=prague` is redundant when the test is already restricted to EOF. You have that in multiple tests.

Reply:
Done